sootup.core/src/main/java/sootup/core/model/SourceType.java

```diff
@@ -23,7 +23,7 @@
  */
 
 public enum SourceType {
-  Phantom,
-  Application,
-  Library
+  Application, // code that we want to analyze e.g. in call graph generation its traversed
+  Library // code that is given but should not be analyzed e.g. in call graph generation its not
```

Comment:
This sounds like an issue for the call graph generation. I think we are not ignoring library Sources

Reply:
noted down as issue #715